weave/trace_server/migrations/002_feedback.up.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the call. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
+
+    /*
+    `call_id`: The unique identifier for the call. This is typically a UUID.
+    */
+    call_id String,
+
+    /*
+    `wb_user_id`: The ID of the user that created the call. This is the ID of the user in the
+    W&B API.
+    */
+    wb_user_id String NULL,
+
+    /*
+    `created_at`: The time that the row was inserted into the database.
+    */
+    created_at DateTime64(3) DEFAULT now64(3),
+
+    /*
+    `feedback_type`: The type of feedback that was given. The prefix "wandb." is reserved for our use.
+    */
+    feedback_type String,
+
+    /*
+    `notes`: An optional comment associated with the feedback.
+    */
+    notes String NULL,
```

Comment:
Both feedback_type and notes here seem to be things i would expect as part of the feedback json itself. Can you share what you have in mind for why these fields should be first-class?

Reply:
I've removed notes, partly because we've decided to have notes as a first-class special W&B feedback type. Feedback type makes sense to me to keep separate because it determines which schema to use to validate the contents of the payload. I'm sure we could make it work, but feels odd to reach into an object to pull out a value that we use to validate the structure of that object.